### ISSUE TITLE
Xamarin.Droid.Sample : Failed at Platform.InitializeAsync

### ISSUE THREAD

**Issue by pnp0a03** [Microsoft/project-rome#17]
Hi,
I've tried to run the xamarin/samples/ConnectedDevices.Xamarin.Droid.Sample.
I could build it and run with my CLIENT_ID. At the first time launch, I've put the my MSA id and pass. But, the app crashed at this point.
It seems that the app crashed at MainActivity.cs, line 63:

`            var result = await Platform.InitializeAsync(this.ApplicationContext, CLIENT_ID);`

I want to debug it but I can't.
Because,  this Platform.InitializeAsync is a part of Xamarin/src/ConnectedDevices.Xamarin.Droid/ , and the source is insufficient to build and debug. It seems that the Jars/connecteddevices-sdk-armv7-release.aar ( you can download it by download-aas.ps1) is old, and it does not contain the definition of AppServiceClientConnectionClosedStatus.  This prevent the build of ConnectedDevices.Xamarin.Droid.

best regards,

**Comment by sphenry** [comment]
Hi @pnp0a03, I've created a new PR that implements the missing methods:
[https://www.nuget.org/packages/Microsoft.ConnectedDevices.Xamarin.Droid/](https://github.com/Microsoft/project-rome/pull/18)
You can try that, or get the Xamarin package from nuget
[https://www.nuget.org/packages/Microsoft.ConnectedDevices.Xamarin.Droid/](https://www.nuget.org/packages/Microsoft.ConnectedDevices.Xamarin.Droid/)

**Comment by pnp0a03** [comment]
Hi sphenry, I could build the Microsoft.ConnectedDevices.Xamarin.Droid 0.4.0 with your PR#18. Thank you !
And, I've noticed - I had always trying to debug it as "Debug" build. But, it seems that .. connecteddevices-sdk-armv7-release.aar is applicable for "Release" build. I had succeeded to run the sample app and enumerate the remote devices with "Release" build. Wao. 
Also I had downloaded the debug version .aar and try it with "Debug" build but this was failed.
Anyway, I could build and run the sample app. Thank you for your help. And, if the debug ver of .aar works, it will help us.
